src/server/graphql/models/CachedUser/cachedUserMutation.js

```diff
@@ -1,11 +1,11 @@
-import r from '../../../database/rethinkdriver';
+import r from '../../../database/rethinkdriver'; // eslint-disable-line id-length
```

Comment:
personally, this looks like a dumb rule that we might want to disable globally? there's plenty of single var imports that we'll probably use in the future...